### PR TITLE
Don't use url-join when uploading to root folder

### PR DESCRIPTION
**Pull request by yfr**
This fixes an error where the upload script creates an '/' folder in root when trying to upload without a remote path.

### CODE REVIEW

**Review by ro-ka — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
+  global:
+    secure: ftlN6d3zrRzL9UDaFILjqu6XEbSQxFjQyuvr1xgxudYtnikblL7u8LkSvbNfd4Gt4MPqaRXnW5hlxY3lPfU1C4dl1nXvoraPElkcVqsxaxbaktwl2b1ECY11VWfbINkr3qIuCAPE+cqEXBQY8rDK/hcYBOpmD0ldqFiEVdxcomE9MwcUSG6+uopTsnNvyDFO/PQSblCDgg86QIZFJxbCSY/AEXJKY2VrF7rQjfIyS1daJtxiTNhrSMOJA4FPFJihUhhhtZouJYbSQJfcGHSEFDYnunGmoTsFpe1qpl7Hrkq/v8sz/bj0ZDB1XrRoaaN07wbcsOnElBZNAtyZo5Mi4hQcElDjWUrUDJLqN9l9jVF9H4TYO8veq8P+zrnLEYj6aLbXDXVYhp/2mRlX9xbKYNrvmTiUzE4HSOIaQQSzTZ5+gKpW9iTaG/Lli7Fo9y229AX8zFXGECwx6nCW51NGBAgPrcM4N1E2F7vRKj739/aLqNUy5A7JNGyCC7k4tQDaFZEqaZG7JbcMFpqcC8cpg6Ap0tISZjIyyg3OSxsj6pT+KCpKRMc0qiTin0t9Cww40q5pd2dbbF0j9odiMYJV8oUAj3AXwXaMxejjnQsX7umcrfRQMTO10WGxe0+dBx06IGEG0EQN50+ymGk83OMe9efJVcF7cA43PH5Ui+4wUa0=
 addons:
   apt:
     sources:
-      - ubuntu-toolchain-r-test
+    - ubuntu-toolchain-r-test
     packages:
-      - g++-4.8
+    - g++-4.8
```
Comment (ro-ka) [inline .travis.yml:21]: Sure about this indention?

File: .travis.yml [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
-
+  - node_modules
 sudo: false
-
 env:
+  matrix:
   - CXX=g++-4.8
-
+  global:
+    secure: ftlN6d3zrRzL9UDaFILjqu6XEbSQxFjQyuvr1xgxudYtnikblL7u8LkSvbNfd4Gt4MPqaRXnW5hlxY3lPfU1C4dl1nXvoraPElkcVqsxaxbaktwl2b1ECY11VWfbINkr3qIuCAPE+cqEXBQY8rDK/hcYBOpmD0ldqFiEVdxcomE9MwcUSG6+uopTsnNvyDFO/PQSblCDgg86QIZFJxbCSY/AEXJKY2VrF7rQjfIyS1daJtxiTNhrSMOJA4FPFJihUhhhtZouJYbSQJfcGHSEFDYnunGmoTsFpe1qpl7Hrkq/v8sz/bj0ZDB1XrRoaaN07wbcsOnElBZNAtyZo5Mi4hQcElDjWUrUDJLqN9l9jVF9H4TYO8veq8P+zrnLEYj6aLbXDXVYhp/2mRlX9xbKYNrvmTiUzE4HSOIaQQSzTZ5+gKpW9iTaG/Lli7Fo9y229AX8zFXGECwx6nCW51NGBAgPrcM4N1E2F7vRKj739/aLqNUy5A7JNGyCC7k4tQDaFZEqaZG7JbcMFpqcC8cpg6Ap0tISZjIyyg3OSxsj6pT+KCpKRMc0qiTin0t9Cww40q5pd2dbbF0j9odiMYJV8oUAj3AXwXaMxejjnQsX7umcrfRQMTO10WGxe0+dBx06IGEG0EQN50+ymGk83OMe9efJVcF7cA43PH5Ui+4wUa0=
```
Comment (ro-ka) [inline .travis.yml]: Should't this better kept private?

**Review by ro-ka — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (yfr) [reply .travis.yml]: This is a encrypted ENV var. That's how travis does it.

File: .travis.yml (hunk 1 above)
Comment (yfr) [reply .travis.yml:21]: Travis formatted it like this.

Comment (ro-ka) [reply .travis.yml:21]: Okay.

**Review by ro-ka — APPROVED** [review]

**Pull request merged**